api.go

```diff
@@ -14,6 +14,28 @@ type APIResponse struct {
 	Message string  `json:"message,omitempty"`
 }
 
+// Identifiable represents a resource with an ID.
+type Identifiable struct {
```

Comment:
Can we make a Flink similar to all the other API wrappers? 
Moreover something like this: 
```
CreateFlinkApplicationRequest{
		Nameable: Nameable{
			Name: name.
		},
	}
```
does not look good  at all.